client/app/scripts/superdesk/notification/tests/notification_spec.js

```diff
@@ -68,3 +68,27 @@ describe('Reload Service', function() {
         expect(reloadService.result.reload).toBe(true);
     }));
 });
+describe('Notify Connection Service', function() {
+    beforeEach(module('superdesk.notification'));
+    beforeEach(module('templates'));
+
+    var rootScope, msg;
+    beforeEach(function() {
+        inject(function($rootScope) {
+            rootScope = $rootScope;
+        });
+    });
+
+    it('can show disconnection message when websocket disconnected', inject(function(notifyConnectionService) {
+        msg = 'websocket connection lost, attempting to reconnect ...';
+        rootScope.$broadcast('disconnected');
+        expect(notifyConnectionService.message).toEqual(msg);
+    }));
+
+    it('can show success message when websocket connected', inject(function(notifyConnectionService) {
+        msg = 'Connected!!!';
+        rootScope.$broadcast('connected');
+        expect(notifyConnectionService.message).toEqual(msg);
+    }));
```

Comment:
As discussed in the office, yes, it is fine. Creating proper unit tests in this would require too many workarounds and code changes to justify the time investment.